.sandbox/manifest.yaml

```diff
@@ -1,5 +1,5 @@
 hooks:
-  build:
+  post-checkout:
```

Comment:
I think place yarn in the build is fine. Generally if code change (including adding application level packages from config file) requires it to run we include it in build so that `cs build` will run it.

Reply:
I see, I'll revert this then.